### ISSUE TITLE
LR decay during the patch-level phase ?

### ISSUE THREAD

**Issue by alxndrTL** [shaochenze/PatchTrain#4]
Hello,

During the patch-level training, is the LR decayed with cosine decay ? Or is it held constant ?
And if it is decayed with cosine, does it fall to the min learning rate right at the end of patch-level training ?

Thank you!

**Comment by shaochenze** [comment]
Yes, in the paper we use the cosine lr decay, and it falls to the min lr at the end. We have also experimented with constant lr afterwards and it also works.
